models/base.rb

```diff
@@ -64,6 +85,15 @@ def self.has_many(name, options = {})
           "#{name[0..-4]}y_ids".to_sym
         end
 
+      case options[:as_json]
+      when :ids
+        json_fields << ids_function_name.to_s
```

Comment:
Is there a specific reason for not using symbol?

Reply:
The hash keys returned by as_json are strings